src/docs/implementations/otlp.adoc

```diff
@@ -0,0 +1,131 @@
+|java
+
+|telemetry.sdk.version
+|<micrometer-core-version> (E.g: 1.11.0)
+
+|service.name
+|unknown_service
+|===
+
+If this config is empty, then resource attributes will be loaded from the environmental variable `OTEL_RESOURCE_ATTRIBUTES`. `service.name` can be overridden by the environmental variable `OTEL_SERVICE_NAME` and this takes precedence over other configs.
+
+== Supported metrics
+https://opentelemetry.io/docs/specs/otel/metrics/data-model/#metric-points[Metric points] define the different data-points that are supported in OTLP. Micrometer supports exporting the below data-points in OTLP format,
+
+1. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#sums[Sums]
+2. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#gauge[Gauge]
+3. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#histogram[Histogram]
+4. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#summary-legacy[Summary]
+
+The below table maps OTLP datapoint and the micrometer meters,
+[%autowidth]
+|===
+|OTLP DataPoint | Micrometer meter type
+
+|Counter, FunctionCounter
+| Sums
+
+|Gauge
+|Gauge
+
+|Histogram
+|Timer, DistributionSummary, LongTaskTimer, FunctionTimer (only sum and count are set)
+
+|Summary
+|Timer, DistributionSummary, LongTaskTimer
+|===
+
+*Note*: `max` on Histogram datapoint is only supported in Delta Aggregation Temporality.
+
+== Histograms and Percentiles
+Micrometer `Timer` and `DistributionSummary` supports configuring http://localhost:3000/docs/concepts#_histograms_and_percentiles[client side percentiles and percentile histogram]. OTLP specification terms Summary Data-point (client-side percentiles) as legacy and not recommended for new applications. Summary data point also cannot have min/max associated with it. Due these reasons micrometer prefers exporting Timers and DistributionSummary as Histogram datapoint. By default a Timer/DistributionSummary without any additional percentile/histogram config will be exported as Histogram Datapoint. However, by configuring the timer to generate only client side percentiles using `publishPercentiles` this can be changed to a Summary data point exporting pre-calculated percentiles. When both `publishPercentiles` and (`publishPercentileHistogram` or `serviceLevelObjectives`) are configured Histogram Datapoint is preferred and pre-calculated percentiles *will not* be generated. See the below table on which data point will be used with different configurations,
+[%autowidth]
+|===
+|Configuration | OTLP Data point
+
+| publishPercentiles
+| Summary
+
+| publishPercentileHistogram
+| Histogram
+
+| serviceLevelObjectives
+| Histogram
+
+| publishPercentiles and (publishPercentileHistogram/serviceLevelObjectives)
+| Histogram
+|===
+
+Alternatively, If you are using spring boot you can use the https://docs.spring.io/spring-boot/docs/3.1.0/reference/htmlsingle/#actuator.metrics.customizing.per-meter-properties[per meter properties] to configure this behaviour.
```

Comment:
I'm thinking if we should use `current` instead of `3.1.0`.
```suggestion
Alternatively, If you are using Spring Boot you can use the https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator.metrics.customizing.per-meter-properties[per meter properties] to configure this behaviour.
```